src/test/java/lotto/model/LottoFactoryTest.java

```diff
@@ -0,0 +1,47 @@
+package lotto.model;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LottoFactoryTest {
+    LottoFactory lottoFactory;
+    @BeforeEach
+    void setUp() {
+        lottoFactory = new LottoFactory();
+    }
+
+    @DisplayName(value = "자동 로또 갯수만큼 생성하기")
+    @Test
+    void create_auto_lottos() {
+        List<Lotto> lottos = lottoFactory.createAutoLottos(3);
+        System.out.println("자동 로또 갯수만큼 생성하기");
+        for(Lotto lotto : lottos) {
+            for(LottoNumber lottoNumber : lotto.getNumbers()) {
+                System.out.print(lottoNumber.toString() + " ");
+            }
+            System.out.println();
+        }
+        System.out.println();
+    }
+
+    @DisplayName(value = "수동 로또 갯수만큼 생성하기")
+    @Test
+    void create_manual_lottos() {
+        List<String> manualNumbers = new ArrayList<>();
+        manualNumbers.add("1, 2, 3, 4, 5, 6");
+        manualNumbers.add("2, 3, 4, 5, 6, 7");
+        manualNumbers.add("3, 4, 5, 6, 7, 8");
```

Comment:
```suggestion
        List<String> manualNumbers = Arrays.asList("1, 2, 3, 4, 5, 6", "2, 3, 4, 5, 6, 7", "3, 4, 5, 6, 7, 8");
```
위와 같이 변경할 수 있을 것 같아요  😁